TF_Tooling_Web/src/Cursors/TouchlessCursor.ts

```diff
@@ -60,18 +60,14 @@ export abstract class TouchlessCursor {
     // Used to make the cursor visible
     ShowCursor(): void {
         this.shouldShow = true;
-        if (this.cursor && this.enabled) {
```

Comment:
this.enabled has been lost when calling SetCursorOpacity

Reply:
Good catch - pushed fix